.github/workflows/delivery-docker.yml

```
@@ -33,29 +33,30 @@ jobs:
             }
 
             return tag.replace(/^v/, '');
-      - name: Checkout workflow
+      - name: Checkout delivery configuration
         uses: actions/checkout@v2
+        with:
+          path: ./main
```

```suggestion
          path: ./delivery
```

Sorry, I'm being a little pedantic here but if `main` is suppose to represent what branch is being checked out then I think that's incorrect. In the GHA UI you are able to specify what branch you would like to run as your `HEAD`. 

I suggested "delivery" but may "head"...? idk.

---

I think head instead, because that's what it's doing - it's the head of the branch you are using